jenkins/scripts/common/colors.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Helper vars for printing in colour. Use with:
```

Comment:
Consistency nit: all the comments in this file use `colour` but the script itself is named `colors`.

Reply:
Good point, done.